variables.css

```diff
@@ -0,0 +1,33 @@
+
+:root {
+    /* CSS HEX */
```

Comment:
Deze variabelen zijn netjes benoemd, maar hieronder wissel je van `text-2-color` (wat is tekst 2?) naar `project-bg-color` en soms zet je er `var` achter. Probeer dit zo veel mogelijk in dezelfde stijl te doen, zoals je hierboven doet :)